src/test/java/teammates/test/cases/util/StringHelperTest.java

```diff
@@ -131,18 +131,36 @@ public void testKeyEncryption() throws InvalidParametersException {
     @Test
     public void testDefaultAesCipherParams() throws Exception {
         //plaintext is less than 1 block long
-        String plaintextLength28 = StringHelper.generateStringOfLength(7, 'A');
-        assertEncryptionUsesExpectedDefaultParams(plaintextLength28);
+        String plaintextLength124 = StringHelper.generateStringOfLength(31, 'A');
+        assertEncryptionUsesExpectedDefaultParams(plaintextLength124);
 
         //plaintext is equal to 1 block
         String plaintextLength128 = StringHelper.generateStringOfLength(32, 'A');
         assertEncryptionUsesExpectedDefaultParams(plaintextLength128);
 
         //plaintext is more than 1 block long
-        String plaintextLength136 = plaintextLength128 + StringHelper.generateStringOfLength(2, 'A');
-        assertEncryptionUsesExpectedDefaultParams(plaintextLength136);
+        String plaintextLength132 = plaintextLength128 + StringHelper.generateStringOfLength(1, 'A');
```

Comment:
Since we are using this method, this can be rewritten as `StringHelper.generateStringOfLength(33, 'A')`